.github/govulncheck/govulncheck-runner.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+# Run govulncheck security scanning on Kata runtime binaries
+
+set -euo pipefail
+
+# Ensure both Go and go-installed tools are in PATH
+export PATH="/usr/local/go/bin:${HOME}/go/bin:${PATH}"
+
+# Change to runtime directory
+cd ../../src/runtime
```

Comment:
This should be done before this script is called.

Reply:
Moved to be before the script is called.